### ISSUE TITLE
Invalid unparse for `Sequel.&(foo: 1)

### ISSUE THREAD

**Issue by backus** [mbj/unparser#278]
**Versions**

```
$ cat Gemfile.lock | grep parser
      parser (~> 3.0.0)
      regexp_parser (~> 2.0, >= 2.0.3)
      unparser (~> 0.6.0)
    parser (3.0.2.0)
    regexp_parser (2.1.1)
    unparser (0.6.0)
      parser (>= 3.0.0)
```

**Reproduction** 

```ruby
2.7.2-mri :001 > source = 'Sequel.&(foo: 1)'
 => "Sequel.&(foo: 1)"

 2.7.2-mri :002 > Unparser.parse(source)
 =>
s(:send,
  s(:const, nil, :Sequel), :&,
  s(:kwargs,
    s(:pair,
      s(:sym, :foo),
      s(:int, 1))))

2.7.2-mri :003 > Unparser.unparse(Unparser.parse(source))
 => "Sequel & foo: 1"

2.7.2-mri :004 > Unparser.parse(Unparser.unparse(Unparser.parse(source)))
Traceback (most recent call last):
       16: from /dev/.rvm/gems/ruby-2.7.2-mri/gems/bundler-2.2.6/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
       15: from /dev/.rvm/gems/ruby-2.7.2-mri/gems/bundler-2.2.6/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
       14: from /dev/.rvm/gems/ruby-2.7.2-mri/gems/bundler-2.2.6/lib/bundler/cli.rb:494:in `exec'
       13: from /dev/.rvm/gems/ruby-2.7.2-mri/gems/bundler-2.2.6/lib/bundler/cli/exec.rb:28:in `run'
       12: from /dev/.rvm/gems/ruby-2.7.2-mri/gems/bundler-2.2.6/lib/bundler/cli/exec.rb:63:in `kernel_load'
       11: from /dev/.rvm/gems/ruby-2.7.2-mri/gems/bundler-2.2.6/lib/bundler/cli/exec.rb:63:in `load'
       10: from /dev/.rvm/rubies/ruby-2.7.2-mri/bin/irb:23:in `<top (required)>'
        9: from /dev/.rvm/rubies/ruby-2.7.2-mri/bin/irb:23:in `load'
        8: from /dev/.rvm/rubies/ruby-2.7.2-mri/lib/ruby/gems/2.7.0/gems/irb-1.2.6/exe/irb:11:in `<top (required)>'
        7: from (irb):4
        6: from /dev/.rvm/gems/ruby-2.7.2-mri/gems/unparser-0.6.0/lib/unparser.rb:106:in `parse'
        5: from /dev/.rvm/gems/ruby-2.7.2-mri/gems/parser-3.0.2.0/lib/parser/base.rb:190:in `parse'
        4: from (eval):3:in `do_parse'
        3: from (eval):3:in `_racc_do_parse_c'
        2: from /dev/.rvm/gems/ruby-2.7.2-mri/gems/parser-3.0.2.0/lib/parser/base.rb:286:in `on_error'
        1: from /dev/.rvm/gems/ruby-2.7.2-mri/gems/parser-3.0.2.0/lib/parser/diagnostic/engine.rb:72:in `process'
Parser::SyntaxError (unexpected token tCOLON)
```

**Comment by mbj** [comment]
interesting one. thx!

**Comment by mbj** [comment]
@backus Released as 0.6.1. Mutant should pick it up nicely.
